proto/pocket/shared/params.proto

```diff
@@ -49,11 +49,18 @@ message Params {
   // application unbonding period will exceed the end of its corresponding proof window close height.
   uint64 application_unbonding_period_sessions = 8 [(gogoproto.jsontag) = "application_unbonding_period_sessions"];
 
-  // The amount of upokt that a compute unit should translate to when settling a session.
+  // The amount of tokens that a compute unit should translate to when settling a session.
+  // It is denominated in fractional uPOKT (1/compute_unit_cost_granularity)
   // DEV_NOTE: This used to be under x/tokenomics but has been moved here to avoid cyclic dependencies.
   uint64 compute_units_to_tokens_multiplier = 9 [(gogoproto.jsontag) = "compute_units_to_tokens_multiplier", (gogoproto.moretags) = "yaml:\"compute_units_to_tokens_multiplier\""];
 
   // gateway_unbonding_period_sessions is the number of sessions that a gateway must wait after
   // unstaking before their staked assets are moved to its account balance.
   uint64 gateway_unbonding_period_sessions = 10 [(gogoproto.jsontag) = "gateway_unbonding_period_sessions"];
-}
+
+  // compute_unit_cost_granularity is the fraction of the base unit (uPOKT) used
```

Comment:
`uPOKT` is the universal "base unit". I get that.

What's not clear to me is how we get ppokt now?